docs/semgrep-app/findings.md

```diff
@@ -0,0 +1,161 @@
+---
+slug: findings 
+append_help_link: true
+title: Findings 
+description: "The Findings page allows users to view, manage, and triage Findings."
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Managing findings in Semgrep App
+
+
+![Screenshot of Semgrep App Findings page ](../img/app-findings-overview.png)<br />
+
+A finding is the core result of Semgrep's analysis. Findings are generated when a Semgrep rule matches a piece of code. 
+
+Types of findings:
+
+* anti-patterns
+* security vulnerabilities, business, or logic bugs
+* audits for dangerous function use
+* reviewing authentication or authorization logic
+* custom pattern matches based on your own custom rules
+
+
+After a finding is generated, developers may:
+
+* View the Semgrep rule and the matching code.
+* Fix the issue detected by the finding.
+* Ignore the finding.
+* Comment on the finding.
+* Create a Jira ticket from the finding (for Enterprise/Team Tier users.)
+
+
+The Semgrep App Finding page displays findings across all projects connected to Semgrep App. It is updated after every scan. Scans are initiated through your CI/CD pipeline, such as GitHub actions. The retention period of these findings varies based on your organization’s tier:
+
+| Retention period | Tier availability |
+| ---------------  | ----------------- |
+| 30-day findings retention | Community (Free) |
+| 1-year findings retention | Team/Enterprise |
+
+## Viewing data in the Findings page
+
+### Navigating to the Findings page
+
+1. Sign in to Semgrep App.
+2. The *Findings* page can be found on the left sidebar.
+
+### Understanding the Findings data
+
+The Findings page displays the following fields: 
+
+| Field       | Description | 
+| ----------  | ------------ | 
+| **Severity**    | The impact of a finding. Possible values: Low, Medium, High,and Critical. |
+| **Finding**     | The file, line, and branch of the pattern match in the code. Clicking on this field brings you to the exact location in the codebase. |
+| **Project**     | The name of the project codebase where the finding was found. Clicking on this field brings you to the project’s URL. |
+| **Rule**        | The name of the rule matched with the code. Clicking on this field brings you to the rule’s entry in the [rule registry](https://semgrep.dev/r). |
+| **Introduced**  | The time when this finding was first found. Clicking on the header row sorts the table by latest or earliest findings. |
+| **Status**      | Refers to a finding’s triage state. |
```

Comment:
I'd replace with "Refers to a finding's state." since it isn't just triage

Reply:
Done!